src/build/languages.js

```diff
@@ -1,5 +1,6 @@
 exports.languages = {
   javascript: 'javascript',
   perl: ['perl', 'perl6'],
-  prolog: 'prolog'
+  prolog: 'prolog',
+  json: 'json'
```

Comment:
This should have been sorted. My bad. Will do in the next PR that adds a `language` support.